SUIApp/SUIApp/ContentView.swift

```diff
@@ -7,16 +7,128 @@
 
 import SwiftUI
 
+/// Экран покупки авто
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let imageNames = ["oka", "urus", "hAven"]
+        static let carEquipments = ["ВАЗ-1111", "Lamborghini", "Hyundai Aventador"]
+        static let carDescriptions = ["Это мой автомобиль", "Это автомобиль Жени", "А это пушка гонка Саши"]
+        static let carDetails = ["Объем 0.7 л \n" +
+                                 "Мощность 33 л.с.\n" +
+                                 "Расход 6,8 л\n" +
+                                 "Объем багажника 210 л",
+                                 "Объем 4.0 л \n" +
+                                 "Мощность 650 л.с.\n" +
+                                 "Расход 12,3 л\n" +
+                                 "Объем багажника 999 л",
+                                 "Объем 999.9 л \n" +
+                                 "Мощность 999 л.с.\n" +
+                                 "Расход 0.01 л\n" +
+                                 "Объем багажника 999 л"]
+        static let notHaveMoneyText = "У тебя нет денег даже на эту"
+        static let howToBuyText = "Хотите приобрести эту бричку?"
+        static let shareTitleText = "Share"
+        static let nonShowDetailText = "Скрыть характеристики"
+        static let showDetailText = "Показать все характеристики"
+        static let emptyText = ""
+        static let twoHundredNumber: CGFloat = 200
+        static let zeroNumber: CGFloat = 0
+    }
+   
+    // MARK: - Public Properrty
+    
     var body: some View {
+        NavigationView {
+            VStack {
+                toggleShow
+                ZStack {
+                    HStack {
+                        Text(carDetail[segmentIndex])
+                            .padding()
+                            .font(.title3)
+                        Spacer()
+                    }
+                    Spacer()
+                    carInfo
+                }
+                buyCar
+            }
+            .navigationTitle(carEquipments[segmentIndex])
+        }
+    }
+    
+    @State private var isToggleOn = false
+    @State private var isSharePresented = false
+    @State private var isAlertShow = false
+    @State private var segmentIndex = 0
+    @State private var textDetail = Constants.emptyText
+    @State private var offset = Constants.zeroNumber
+    private var imageName = Constants.imageNames
+    private var carEquipments = Constants.carEquipments
+    private var carDescription = Constants.carDescriptions
+    private var carDetail = Constants.carDetails
+    
+    private var buyCar: some View {
```

Comment:
```suggestion
    private var buyCarView: some View {
```
на конце View где возвращаем some View 
и ниже также